app/assets/javascripts/contacts/show.js

```diff
@@ -0,0 +1,60 @@
+(function($){
+  var accepts = "application/vnd.addressbook_service+json; version=1";
+  var async = true;
+  var contentType = "application/json";
+  var url = "http://localhost:5100/api/contacts";
+  var storedContacts = {};
+
+  var erredContacts = function(xhr, status, error) {
+    console.log("retrieveContacts had an error");
+    console.log(error);
+  };
+
+  var processContacts = function(contacts, status, xhr) {
+    storedContacts = contacts;
+
+    for (var i = 0; i < storedContacts.length; i++) {
+      $(".contacts").append(
+        $(
+          "<li><a data-role='contact' data-value='" + i + "' href='#'>" +
+          storedContacts[i].name + "</a></li>"
+        );
+      );
```

Comment:
Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.